website/docs/plugin/log/managing.mdx

```diff
@@ -25,6 +25,7 @@ The following environment variables turn on logging and can help you filter log
 | `TF_LOG_PROVIDER_{PROVIDER_NAME}` (e.g., `TF_LOG_PROVIDER_AWS`)| Specific provider  | Only applies to providers that write logs using the `tflog` package. Does not affect logs written by legacy `log.Print*` functions. |
 |`TF_LOG_SDK` | All SDKs for providers used during the run | Only applies to logs written with `terraform-plugin-log`. For remaining SDKV2 logs, use `TF_LOG` or `TF_LOG_PROVIDER`. |
 | `TF_LOG_SDK_PROTO` | Protocol layer logging for providers built on [`terraform-plugin-go`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-go). | Overrides `TF_LOG_SDK`. |
+| `TF_LOG_SDK_FRAMEWORK`| [`terraform-plugin-framework`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema) | Overrides `TF_LOG_SDK`. |
```

Comment:
Was this URL meant to be: https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework ?

Reply:
ha, yes it was 😄 

```suggestion
| `TF_LOG_SDK_FRAMEWORK`| [`terraform-plugin-framework`](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework) | Overrides `TF_LOG_SDK`. |
```